src/client/app/app.js

```diff
@@ -8,8 +13,25 @@ import styles from './app.less';
 var App = React.createClass({
   render: function () {
     return (
-      <SearchPage />
-    )
+      <Router>
+        <Switch>
+          <Route
+            exact
+            path="/"
+            component={SearchPage}
+          />
+          <Route
+            path="/search/:searchQuery?"
```

Comment:
Why do you need question mark at the end of the path?

Reply:
it makes the parameter is optional